lib/mondrian/olap/connection.rb

```diff
@@ -339,7 +339,9 @@ def jdbc_uri
         when 'sqlserver'
           uri = "jdbc:sqlserver://#{@params[:host]}#{@params[:port] && ":#{@params[:port]}"}"
           uri << ";databaseName=#{@params[:database]}" if @params[:database]
-          uri << ";integratedSecurity=#{@params[:integrated_security]}" if @params[:integrated_security]
+          integrated_security = @params[:integrated_security] || @params.dig(:properties, 'integratedSecurity')
```

Comment:
Do not use `dig` as it is not present in JRuby 1.7 which is still supported by `mondrian-olap`.
Also better support just `@params[:integrated_security]` as we do not use `@params[:properties]` for anything else in case of `sqlserver`.